testsuite/analyze_itensor.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import print_function
+import sys
+import unittest as ut
+import numpy as np
+import espressomd
+
+
+class AnalyzeITensor(ut.TestCase):
+    """Test the inertia tensor analysis"""
+    
+    system = espressomd.System()
+
+    @classmethod
+    def setUpClass(self):
+        box_l = 50.0
+        self.system.box_l = [box_l, box_l, box_l]
+        self.system.cell_system.skin = 0.4
+        self.system.time_step = 0.01
+        self.system.thermostat.turn_off()
+        for i in range(10):
+            self.system.part.add(id=i, pos=np.random.random(3) * box_l,type=0)
```

Comment:
shouldnt we try to keep tests deterministic, i.e. setting a fixed seed for numpy random?

Reply:
resolved by offline discussion: this test should pass for all given positions therefore it does not harm to use random positions